static/src/js/containers/MapContainer/MapContainer.tsx

```diff
@@ -6,27 +6,41 @@ import React, {
   useLayoutEffect
 } from 'react'
 import { connect } from 'react-redux'
-import PropTypes from 'prop-types'
+import { Dispatch } from 'redux'
 import { difference } from 'lodash-es'
+import { Geometry } from 'ol/geom'
 
+// @ts-expect-error The file does not have types
 import actions from '../../actions'
+// @ts-expect-error The file does not have types
 import { metricsMap } from '../../middleware/metrics/actions'
 
 import { eventEmitter } from '../../events/events'
+// @ts-expect-error The file does not have types
 import { getFocusedCollectionGranuleResults } from '../../selectors/collectionResults'
+// @ts-expect-error The file does not have types
 import { getFocusedCollectionId } from '../../selectors/focusedCollection'
+// @ts-expect-error The file does not have types
 import { getColormapsMetadata } from '../../selectors/colormapsMetadata'
+// @ts-expect-error The file does not have types
 import { getFocusedGranuleId } from '../../selectors/focusedGranule'
+// @ts-expect-error The file does not have types
 import { getGranulesMetadata } from '../../selectors/granuleMetadata'
+// @ts-expect-error The file does not have types
 import { getMapPreferences } from '../../selectors/preferences'
+
 import { isPath } from '../../util/isPath'
-import { locationPropType } from '../../util/propTypes/location'
-import projectionCodes from '../../constants/projectionCodes'
 import { projectionConfigs } from '../../util/map/crs'
+// @ts-expect-error The file does not have types
 import murmurhash3 from '../../util/murmurhash3'
 import hasGibsLayerForProjection from '../../util/hasGibsLayerForProjection'
+
+// TODO convert this file to typescript? lambda code needs to work with it
```

Comment:
Remove? This file is ts now?

Reply:
It was referencing the sharedUtils/tags file. I don't want to get into TS in lambdas in this ticket